dependencies.yaml

```diff
@@ -30,24 +29,27 @@ files:
     output: none
     includes:
       - cuda_version
+      - depends_on_libcuspatial
       - test_libcuspatial
   test_python:
     output: none
     includes:
       - cuda_version
+      - depends_on_cuproj
+      - depends_on_cuspatial
       - py_version
       - test_python_cuspatial
       - test_python_cuproj
-      - test_cuspatial
```

Comment:
Notice that here, I've taken a list that previously contained `cuproj`, `cuspatial`, and `libcuspatial` with only `cuproj` and `cuspatial`.

That's intentional.... this `test_python` list is used for testing the Python conda packages:

https://github.com/rapidsai/cuspatial/blob/1544e7bd16f11457436d1fb6a43da0db3aa9af57/ci/test_python.sh#L13-L17

Omitting `libcuspatial` here gives us a chance to learn about packaging errors of the form "`cuspatial` is missing a dependency on `libcuspatial" in CI.